### ISSUE TITLE
Shell script task cancels after 60 minutes on Ubuntu regardless of timeout setting

### ISSUE THREAD

**Issue by dcatonR1** [Microsoft/vsts-agent#747]
## **Agent version and platform**
Version of your agent? 2.105.7<br>
OS of the machine running the agent? Ubuntu 14.02
<br>
## **VSTS type and version**
On-Prem TFS 2017 RTM
<br>
If On-Prem TFS, which release, 2015 RTM/QU1/QU2?

## **What's not working?** 
As part of a build definition I have a shell script task.  The script runs 'make' with one argument.  The make takes quite a long time to run (well over an hour).  I have the task timeout set to 0, which is supposed to mean no timeout, but it is timing out after 60 minutes.  I set the timeout to 180, and it still times out after 60 minutes.

Is this a known issue?  Is this even the right place?  The home page for this project is a bit unclear as to whether this is the build agent included in TFS 2017.
## **Agent and Worker's diag log**

[Worker_20170110-050011-utc.zip](https://github.com/Microsoft/vsts-agent/files/697184/Worker_20170110-050011-utc.zip)

**Comment by ericsciple** [comment]
Yes this is the agent repo, but the issue may be server side. I briefly looked through the worker diag log and it doesn't look like the timeout comes down in the job message. I see the worker is getting a cancellation message from the agent (which i'm assuming is getting a message from the server, but haven't made it that far in my investigation). @TingluoHuang knows the details offhand. Otherwise I can dig in more later.

**Comment by ericsciple** [comment]
ah, and i had forgotten about the individual task timeout :)
